### PR TITLE
Fix https://github.com/NuGet/Home/issues/1468

### PR DESCRIPTION
**Pull request by yishaigalatzer**
Private PR for @emgarten 

Remove duplicate RID graphs, this not only fixes the exception, it also prevents unnecessary restores

### CODE REVIEW

File: src/NuGet.Core/NuGet.Commands/RestoreCommand.cs [hunk 1]
```diff
@@ -289,14 +292,18 @@ public Task<RestoreResult> ExecuteAsync()
 
             foreach (var pair in runtimesByFramework)
             {
-                _log.LogVerbose(Strings.FormatLog_RestoringPackages(pair.Key.DotNetFrameworkName));
+                // Remove duplicate pairs
+                if (existingGraphs.Any(g => string.IsNullOrEmpty(g.RuntimeIdentifier) && g.Framework == pair.Key))
```
Comment (yishaigalatzer) [inline src/NuGet.Core/NuGet.Commands/RestoreCommand.cs:296]: Looking at this again, this looks wrong

**Comment by MeniZalzman** [comment]
@yishaigalatzer is this PR ready?

**Comment by yishaigalatzer** [comment]
No. Justin needs to take over

**Comment by MeniZalzman** [comment]
:watch: @emgarten please work on this PR with Yishai

**Comment by yishaigalatzer** [comment]
Fixed by @emgarten

**Pull request closed without merging**